shared/src/commonMain/kotlin/co/nimblehq/kmm/showcase/data/JobDetails.kt

```diff
@@ -0,0 +1,100 @@
+package co.nimblehq.kmm.showcase.data
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Data(
+    @SerialName("data") val data: JobDetails
+)
+
+@Serializable
+data class JobDetails(
+    @SerialName("id") val id: String? = null,
+    @SerialName("confirmed_at") val confirmedAt: Int? = null,
+    @SerialName("created_at") val createdAt: Int? = null,
+    @SerialName("expire_at") val expireAt: Int? = null,
+    @SerialName("maximum_price") val maximumPrice: Int? = null,
+    @SerialName("message") val message: String? = null,
+    @SerialName("minimum_price") val minimumPrice: Int? = null,
+    @SerialName("proposed_at") val proposedAt: String? = null,
+    @SerialName("status") val status: String? = null,
+    @SerialName("updated_at") val updatedAt: Int? = null,
+    @SerialName("farmer") val farmer: Farmer? = Farmer(),
+    @SerialName("job_substance_type") val jobSubstanceType: JobSubstanceType? = JobSubstanceType(),
+    @SerialName("plot") val plot: Plot? = Plot(),
+    @SerialName("time_slot") val timeSlot: TimeSlot? = TimeSlot(),
+    @SerialName("weather_forecast") val weatherForecast: WeatherForecast? = WeatherForecast()
+)
+
+@Serializable
+data class Farmer(
+    @SerialName("id") val id: String? = null,
+    @SerialName("email") val email: String? = null,
+    @SerialName("first_name") val firstName: String? = null,
+    @SerialName("last_name") val lastName: String? = null,
+    @SerialName("line_id") val lineId: String? = null,
+    @SerialName("phone_number") val phoneNumber: String? = null,
+    @SerialName("profile_picture") val profilePicture: String? = null
+)
+
+@Serializable
+data class SubstanceType(
+    @SerialName("type") val type: String? = null,
```

Comment:
we don't need this, it's from JSONAPI

Reply:
I am generating POJO from JSON response. If so please remove unnecessary fields in JSON response if we don't need them.